debian/eos-paygd.install

```diff
@@ -1,6 +1,7 @@
 lib/systemd/system/eos-paygd.service
 lib/systemd/system/eos-paygd-2.service
 usr/lib/tmpfiles.d/eos-paygd.conf
+usr/lib/*/eos-payg-1/libeos-payg-1.a
```

Comment:
This should be part of the `libeos-payg-1-dev` package, so listed in `libeos-payg-1-dev.install`. It doesn't need to be installed on end-user systems, which `eos-paygd` is.

Reply:
ya! sure!  fixed in the new commit.